src/analysis/analysis-poller.js

```diff
@@ -1,22 +1,32 @@
 var _ = require('underscore');
 var BackbonePoller = require('backbone-poller');
 
-var AnalysisPoller = function () {};
+var AnalysisPoller = function () {
```

Comment:
I prefer to use [function declarations](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/function) instead [function expresions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/function) to keep anonymous functions away from the stack traces.

Reply:
👍